libbeat/docs/processors-using.asciidoc

```diff
@@ -32,6 +32,20 @@ condition is present, then the action is executed only if the condition is
 fulfilled. If no condition is passed, then the action is always executed.
 * `<parameters>` is the list of parameters to pass to the processor.
 
+
+You can also scope processors to individual plugin prospectors by placing them under the prospector definition, this is common when wanting to process only logs from modules such as docker, but having more than one prospector enabled: 
```

Comment:
Thanks for your contribution!

In 6.3, we are renaming `prospectors` to `inputs`. Both names are valid in the config, but since the docs reflect the name change, we should  use `input` here. I'd also prefer to simplify the language (we're getting users from all sorts of backgrounds these days). Because this content is shared across all the Beats books, we also need to add some conditional coding so that the info renders in the Filebeat docs, but not elsewhere. 

I'd replace your changes with the following text (includes asciidoc formatting to make everything work):

```
ifeval::["{beatname_lc}"=="filebeat"]
You can define processors at the top-level in the config to filter all data
collected by {beatname_uc}. Or you can specify processors under a specific input
to filter the data collected by that input. For example:

[source,yaml]
------
- type: <input_type>
  processors:
   - <processor_name>:
     when:
        <condition>
     <parameters>
...
------

This is useful when you have more than one input defined, but want to filter
only the events collected by a specific input.
endif::[]
```

I wouldn't mention modules here because that's a specific (more advanced) use case.